reference/4.0/Microsoft.PowerShell.Management/Get-HotFix.md

```diff
@@ -146,8 +146,9 @@ This cmdlet supports the common parameters: -Debug, -ErrorAction, -ErrorVariable
 
 ## INPUTS
 
-### None
-You cannot pipe input to Get-HotFix.
+### String
+
+You can pipe one or more computer names to Get-HotFix.
 
 ## OUTPUTS
```

Comment:
Remove 'Outputs' header to be consistent with changes made to 5/5.1 versions of article.

Reply:
fixed output section in v5/5.1